doc/org.eclipse.cdt.doc.user/src/getting_started/cbs_launchbar.adoc

```diff
@@ -0,0 +1,52 @@
+////
+Copyright (c) 2000, 2025 Contributors to the Eclipse Foundation
+This program and the accompanying materials
+are made available under the terms of the Eclipse Public License 2.0
+which accompanies this distribution, and is available at
+https://www.eclipse.org/legal/epl-2.0/
+
+SPDX-License-Identifier: EPL-2.0
+////
+
+// pull in shared headers, footers, etc
+:docinfo: shared
+
+// support image rendering and table of contents within GitHub
+ifdef::env-github[]
+:imagesdir: ../../images
+:toc:
+:toc-placement!:
+endif::[]
+
+// enable support for button, menu and keyboard macros
+:experimental:
+
+// Until ENDOFHEADER the content must match adoc-headers.txt for consistency,
+// this is checked by the build in do_generate_asciidoc.sh, which also ensures
+// that the checked in html is up to date.
+// do_generate_asciidoc.sh can also be used to apply this header to all the
+// adoc files.
+// ENDOFHEADER
+
+== Launch Bar
+
+CBS projects rely on the *LaunchBar*. Make sure the launch bar plugin
+is installed and enabled. In the global preferences the launch bar can
+be enabled or disabled.
+
+image:launchbar_preferences.png[Launch Bar preferences]
+
+The launch bar exists out of several components:
+
+image:launchbar.png[Launch Bar]
+
+. Build button, to launch a build.
+. Run button, to launch a run. This button will change to a Debug
```

Comment:
Should we mention profile mode somewhere here too?

Reply:
I was doubting about mentioning it in "getting started". The Profile mode is not in the launch mode selector before you run profiling. Profiling only works properly in Makefile projects, not in CMake. And you need to have the profiling plugin installed before it is available at all. I think it is more an advanced topic.